proposed/extended-coding-style-guide.md

```diff
@@ -72,7 +81,7 @@ Code MUST follow all rules outlined in [PSR-1].
 
 All PHP files MUST use the Unix LF (linefeed) line ending.
 
-All PHP files MUST end with a single blank line.
+All PHP files MUST end with a single containing only a single newline (LF) character.
```

Comment:
This is so ridiculously verbose, people should just know this stuff.

Could you open an issue please?